docs/UserGuide.md

```diff
@@ -385,6 +393,49 @@ Examples:
 
 <br>
 
+### Editing a patient’s multi-valued attributes
```

Comment:
Actually, a concern I have is that this section may appear to be repetitive, since the commands for the individual features are already explained afterwards